client/state/selectors/is-eligible-for-domain-to-paid-plan-upsell.js

```diff
@@ -1,11 +1,9 @@
 /**
  * Internal dependencies
  */
-import {
-	canCurrentUser,
-	isMappedDomainSite,
-	isSiteOnFreePlan,
-} from 'state/selectors/';
+import canCurrentUser from './can-current-user';
+import isMappedDomainSite from './is-mapped-domain-site';
+import isSiteOnFreePlan from './is-site-on-free-plan';
 
```

Comment:
I think if we use strings like `state/selectors/is-mapped-domain-site` in `registerMock()` we can go back to the normal style